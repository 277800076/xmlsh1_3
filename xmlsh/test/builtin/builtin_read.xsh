# builtin read command
# reads input into string variables

read a <<EOF
foo
EOF

echo a is $a

read a b c <<EOF
foo bar spam
EOF

echo a b c is $a $b $c
