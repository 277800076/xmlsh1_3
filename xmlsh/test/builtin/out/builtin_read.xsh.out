a is foo
a b c is foo bar spam
